OloEngine/src/OloEngine/Scripting/C#/ScriptEngine.cpp

```
@@ -147,6 +147,7 @@ namespace OloEngine
     };
 
     static ScriptEngineData* s_Data = nullptr;
+    static Scene* s_TestSceneContext = nullptr;
```

Clang (14.0.6)

[warning] variable 's_TestSceneContext' is non-const and globally accessible, consider making it const

(cppcoreguidelines-avoid-non-const-global-variables)